src/tests/JIT/HardwareIntrinsics/Arm/Shared/Helpers.cs

```diff
@@ -1769,6 +1803,8 @@ private static byte HighNarrowing(ushort op1, bool round)
 
         public static ushort AddWidening(ushort op1, byte op2) => (ushort)(op1 + op2);
 
+        public static ulong AddWidening(ulong op1, byte op2) => (ulong)(op1 + (ulong)op2);
```

Comment:
do we also not need the following?
```
        public static uint AddWidening(uint op1, byte op2) => (uint)(op1 + (uint)op2);
```

Reply:
Not for the testing. `Sve.AddAcross()` always widens to 64bits regardless of the input.

`helper.cs` seems to be taking the approach of only adding helper functions when they are needed.